frontend/frontend.py

```diff
@@ -62,9 +63,14 @@ def draw_gradio_ui(opt, img2img=lambda x: x, txt2img=lambda x: x,imgproc=lambda
                                     _js='(x) => navigator.clipboard.writeText(x)', fn=None, show_progress=False)
                             output_txt2img_stats = gr.HTML(label='Stats')
                     with gr.Column():
+                        with gr.Row():
```

Comment:
Big no on this, this complicates the UI and can be done with dynamic params (which needs documentation)

Reply:
I agree that it isn't the best looking solution (although with the added styles.css it looked much worse lol), and while I agree dynamic params are powerful, I don't think they're really a replacment for just a +- button, reason is, if you're doing dynamic params you're treating the UI more like CLI, which in that case you're probably trying to minimize button presses etc, to me tho, even if I can do CLI I'd rather use the buttons (I like to just use a mouse while I generate hundreds of photos lol) so I'd rather use buttons.

Not saying which way is correct or whether we should or shouldn't remove it, I just think this should be an option, other users that saw this already asked for it to be also added to the CFG parameter cause like the steps, it's hard to control.

Maybe we change this into an option and set the non-button slider as the default?

Reply:
dynamic params isn't the best solution for sure, but gradio is restrictive, we use sliders because for most things they are most useful. I wish they had an option to arbitrarily adding params, but they don't 
Adding buttons like this (outside of the bounding box of the slider) is just not looking great and is not a great solution, as I already saw the same users ask for this everywhere.

This tab has a nice looking ui, and it's consistent, I would like to keep it that way

Reply:
> dynamic params isn't the best solution for sure, but gradio is restrictive, we use sliders because for most things they are most useful. I wish they had an option to arbitrarily adding params, but they don't Adding buttons like this (outside of the bounding box of the slider) is just not looking great and is not a great solution, as I already saw the same users ask for this everywhere.
> 
> This tab has a nice looking ui, and it's consistent, I would like to keep it that way

It's the same old question of design vs function I guess, you sure we shouldn't just add it as an option and not set it to default?

Reply:
I do agree having the controls lay outside the bounding box is an inconsistent look. Couldn't the "-" and "+" be placed inside of the bounding box, possibly at the beginning and end of the slider bar itself? That would make it more consistent with the UI and also make it more obvious what they do. 

Or is that just not something Gradio will allow due to how it functions?

Reply:
unfortunately that isn't an option, if it was I'd implemented like that haha